### ISSUE TITLE
regression: Widespread Nighlty failures around introspection queries

### ISSUE THREAD

**Issue by philip-stoev** [MaterializeInc/materialize#11431]
### What version of Materialize are you using?

803beb013ba34b07046964b31c176b7d0a1b17da

### How did you install Materialize?

Docker image

### What was the issue?

The Nightly is failing repeatably across the board for --workers 1, --workers 32 and other builds. The commonality between the failures is that queries against the mz_* tables fail to return the expected result within 30 seconds. Upping the timeout to 60 seconds did not seem to fix the problem.

The issue is readily repeatable in the Nightly build and not really repeatable outside of it, even when using EC2 .

**Comment by philip-stoev** [comment]
Git  bisect reports the following:

```
bef493d353907fe75fb71dd3bd36e2e0e8c6d689 is the first bad commit
commit bef493d353907fe75fb71dd3bd36e2e0e8c6d689
Merge: 8bf896c89 11c923c74
Author: Nikhil Benesch <nikhil.benesch@gmail.com>
Date:   Thu Mar 24 12:03:27 2022 -0400

    Merge pull request #11335 from frankmcsherry/replicated_client
    
    Restartable, replicated client

 src/dataflow-types/src/client.rs            | 250 +++++++++++++++++++++++++---
 src/dataflow-types/src/client/controller.rs |   6 +-
 src/dataflow-types/src/client/replicated.rs | 249 +++++++++++++++++++++++++++
 3 files changed, 483 insertions(+), 22 deletions(-)
```

The revision immediately prior to the one above does indeed work. git bisect log:

```
git bisect start '--first-parent'
# good: [4b2a0fb7527c2573e4ea75e1c9d285a291f7e494] doc: add CrunchyBridge to supported tools (#11364)
git bisect good 4b2a0fb7527c2573e4ea75e1c9d285a291f7e494
# bad: [803beb013ba34b07046964b31c176b7d0a1b17da] Merge pull request #11391 from philip-stoev/stabilize-cluster-test
git bisect bad 803beb013ba34b07046964b31c176b7d0a1b17da
# bad: [1976bc100a9e49dc0b2f140771c0cb701469e35d] Merge pull request #11383 from benesch/tcp-retries
git bisect bad 1976bc100a9e49dc0b2f140771c0cb701469e35d
# bad: [22398ed3fc57d4890bcc88445f4f6ff03270e213] doc/user: another example about how to use date_trunc to create windows buckets (#11282)
git bisect bad 22398ed3fc57d4890bcc88445f4f6ff03270e213
# bad: [bef493d353907fe75fb71dd3bd36e2e0e8c6d689] Merge pull request #11335 from frankmcsherry/replicated_client
git bisect bad bef493d353907fe75fb71dd3bd36e2e0e8c6d689
# good: [8bf896c8920f702258b8d1b355d4249aa3ac821c] Merge pull request #11339 from benesch/orchestrator
git bisect good 8bf896c8920f702258b8d1b355d4249aa3ac821c
# first bad commit: [bef493d353907fe75fb71dd3bd36e2e0e8c6d689] Merge pull request #11335 from frankmcsherry/replicated_client
```

**Comment by philip-stoev** [comment]
@frankmcsherry FYI

**Comment by philip-stoev** [comment]
To reproduce, run a manual Nightly build. Put the full git SHA that you want to test in the 'Revision' box, and a bogus name in the Branch box. Once the build starts, it will hang waiting for manual selection .  Select "workers-1" and let it continue.

**Comment by antiguru** [comment]
Fixed in #11436
